racetrack_commons/racetrack_commons/plugin/loader.py

```diff
@@ -51,14 +49,19 @@ def load_plugin_from_zip(plugin_zip_path: Path) -> PluginData:
         assert plugin_zip_path.is_file(), f'no such file {plugin_zip_path}'
         
         extracted_plugins_dir = plugin_zip_path.parent / EXTRACTED_PLUGINS_DIR
-        if not extracted_plugins_dir.is_dir():
-            extracted_plugins_dir.mkdir(parents=True, exist_ok=True)
+        _ensure_dir_exists(extracted_plugins_dir)
+        extracted_plugin_path = extracted_plugins_dir / plugin_zip_path.stem
+
+        plugins_config_dir = plugin_zip_path.parent / PLUGINS_CONFIG_DIR
+        _ensure_dir_exists(plugins_config_dir)
+        _ensure_dir_exists(plugins_config_dir / plugin_zip_path.stem)
```

Comment:
Because inside _ensure_dir_extists there's parents=True, just the second call will be enough.

Reply:
Unfortunately, both are needed to set the proper "chmod" (777) of these directories. I've run into the issues, when I did just the second line, then the  `plugins_config_dir` had 700 mode, as far as I remember.